Array or List binding over the FormData #78

emrekara37 opened the pull request:
Hi,
#78 
I tried with the data in this issue it working

dj-nitehawk commented:
so, for it to work, the incoming data should be a string containing a json object array right?
also what if we changed the match condition to `IEnumerable` instead of `IList`? would that cover a larger range of property types?

emrekara37 commented:
> so, for it to work, the incoming data should be a string containing a json object array right? also what if we changed the match condition to `IEnumerable` instead of `IList`? would that cover a larger range of property types?


Yes, For example this issue #78  

```csharp
var cars= Form["Cars"] // {"Mileage":1,"LicensePlate":"A"} , {"Mileage":2,"LicensePlate":"B"} , {"Mileage":3,"LicensePlate":"C"}
```

And i added just  **[  ]** 


Yes i think we can change IEnumerable instead of IList,

@dj-nitehawk

dj-nitehawk commented:
great! thank you for the contribution 🙏🙏🙏
